.prettierrc.js

```diff
@@ -1,17 +1,16 @@
 module.exports = {
+  // NOTE: This is actually the default value but it is being overwritten by
+  // the solidity plugin somehow.
   bracketSpacing: true,
-  printWidth: 129,
```

Comment:
Remove the current print width and set it to default of 80. `prettier` has a section on its rational for using 80 character width [here](https://prettier.io/docs/en/options.html#print-width) and [here](https://prettier.io/docs/en/rationale.html#print-width).

Reply:
Will this affect the smart contract of just the JS? I am ok with the JS, but not the smart contract.

Reply:
The second link says something about prettier sometimes ignoring lines. I was not aware of this without having to explicitly declare a line to be ignored which I find ugly.

Reply:
From what I understand it has specific logic for imports and unit tests, but for other things you still need to manually ignore a line.

Reply:
Also this change only affects TS moving forward, JS and Sol should remain unchanged.